beacon_node/eth2_libp2p/src/rpc/mod.rs

```diff
@@ -97,7 +102,25 @@ pub struct RPC<TSpec: EthSpec> {
 impl<TSpec: EthSpec> RPC<TSpec> {
     pub fn new(log: slog::Logger) -> Self {
         let log = log.new(o!("service" => "libp2p_rpc"));
+        let limiter = RateLimiterBuilder::new()
+            .one_every(Protocol::MetaData, Duration::from_secs(5))
+            .one_every(Protocol::Ping, Duration::from_secs(15))
+            .one_every(Protocol::Status, Duration::from_secs(1))
```

Comment:
```suggestion
            .n_every(Protocol::Status, 3, Duration::from_secs(15))
```
Hopefully we dont get status'd too often. We sometimes do double status's so maybe we'll allow some burst for status